### PR TITLE
[HW-3] - Angular Material

### PR DESCRIPTION
**Pull request by Oleksandr96**
- Angular Material components
- Post form validation fix
- Code prettify

Screenshots: 
image

image

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
+    BlogModule,
+    FormsModule,
+    ReactiveFormsModule,
+    BrowserAnimationsModule,
+    MatDialogModule,
+    MatButtonModule,
+    MatFormFieldModule,
+    MatInputModule,
+    MatIconModule,
+    MatToolbarModule,
```
Comment (MMaksymivUa) [inline src/app/app.module.ts:30]: create shared/material module

File: src/app/blog/blog.module.ts [hunk 2]
```diff
@@ -5,10 +5,20 @@ import { FormsModule, ReactiveFormsModule } from '@angular/forms';
...
 @NgModule({
   declarations: [PostComponent, SidebarComponent, BlogComponent],
-  imports: [FormsModule, ReactiveFormsModule, BrowserModule],
+  imports: [
+    FormsModule,
+    ReactiveFormsModule,
+    BrowserModule,
+    MatCardModule,
+    MatButtonModule,
+    MatIconModule,
```
Comment (MMaksymivUa) [inline src/app/blog/blog.module.ts:20]: import shared/material module

File: src/app/components/post-form/post-form.component.html [hunk 3]
```diff
@@ -1,44 +1,77 @@
...
+            cdkFocusInitial
+            [disabled]="newPostForm.invalid">Save post
+    </button>
+
+  </form>
+</div>
+
+
+
+
```
Comment (MMaksymivUa) [inline src/app/components/post-form/post-form.component.html:77]: formatting

Comment (Oleksandr96) [reply src/app/components/post-form/post-form.component.html:77]: fixed

File: src/app/app.module.ts (hunk 1 above)
Comment (Oleksandr96) [reply src/app/app.module.ts]: fixed

File: src/app/blog/blog.module.ts (hunk 2 above)
Comment (Oleksandr96) [reply src/app/blog/blog.module.ts]: fixed

**Pull request merged**